Tickets/preops 3838 (#32)
lsst/rtn-011: commissioning.tex

\section{Rubin Observatory Commissioning}
\label{sec:commissioning}

\subsection{Commissioning Milestones}
\label{ssec:commissioning-milestones}

> Commissioning work is being planned around two major milestones, ``First Photon'' and ``System First Light''. 

Reviewer: This is more complicated with ComCam. If First photon is for LSTCam, then the period between FP and FL is less than 2 months (see dmtn-232). The text below needs to be reconciled with ComCam. It may be the easiest thing to do is not talk about first photon but just the activities.

Author: Added in the ComCam FirstLight milestone

> \textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  focal plane to produce an image of the night sky.

Reviewer: Formally only defined for LSSTCam.

Reviewer 2: I think it would assist users' comprehension if we just started saying "LSSTCam First Photon", throughout, then.  In particular that will help users understand that the 1.5-month period you suggest below doesn't include the time during which we'll be tuning up the AOS on ComCam.

Author: I originally understood "First Photon" to refer to the first photons through the optical system, irrespective of which camera they fell on. But I'm not sure now. If "First Photon" is to refer only to the first photons through the optical system and imaged by LSSTCam then we need another milestone ComCam on sky, as  a lot of the work that would be done after the original "LSSTCam First Photon" now gets done with ComCam, resulting in the shorter time between FP and SFL (as Gregory points out)

Author: All definitions now updated

> \textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  focal plane to produce an image of the night sky.

Reviewer: ```suggestion
\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  LSSTCam focal plane to produce an image of the night sky.
```

Author: All definitions updated

\textbf {System First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the full focal plane (LSSTCam) and have a well understood technical path towards meeting the Construction Completeness criteria   \citeds{sitcomtn-061}.
Also referred to as \textbf{First Light}. 
These images will be shared with the worldwide community. 

First Photon occurs following the successful completion of system integration. 
There are no quality criteria applied to the images produced for the First Photon milestone. 
First Photon  marks the start of the the on-sky engineering phase.
System First Light  marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases of commissioning.

> The time from First Photon to System First Light is anticipated to be three months.

Reviewer: If you change 3 months to 1.5 months, then maybe this is all ok.

Author: Updated paragraph

> The time from First Photon to System First Light is anticipated to be three months.

Reviewer: ```suggestion
The time from First Photon to System First Light is anticipated to be 1.5 months.
```

Author: Removed - we now have > 1 "First Photon"

\label{ssec:commissioning-schedule}

Following the October 2023 project schedule workshop, the Rubin Construction Project is re-optimizing the sequence of integration activities in light of recent subcomponent delays, in particular, (i) delay in shipment of the Camera and (ii) necessary repairs to the  summit dome crane.
If current estimates hold, it will be beneficial to re-implement on-sky data-taking with ComCam (previously removed from the schedule in order to install LSSTCam sooner).
The updated plan calls for on-sky data to be taken with ComCam for approximately two months, around July-August 2024 to support Telescope commissioning ,primarily the Active Optics System. 
This is approximately four months earlier than could be done with LSSTCam. 
The Project will make the final, confirming decision about ComCam on-sky data-taking in February 2024.

\begin{figure}[htb]
\centering
\includegraphics[width=0.98\linewidth]{figures/rubinobs_on-sky_commissioning_and_early_science.pdf}
\caption{Detailed schedule of commissioning  and early science activities relative to System First Light, as of October 2023.}
\label{fig:commissioning-es-schedule}
\vspace{0.1cm}
\end{figure}

> Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.

Reviewer: The first block is inconsistent with ComCam. It is 6+months from ComCam on sky to FL. Or if this means FP with LSSTCam then that block is only 6 weeks. See dmtn232.

> System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 2 months after on sky commissioning activities begin.

Reviewer: ```suggestion
System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 1.5 months after on sky commissioning activities begin with LSSTCam.
```

\section{Rubin Observatory Commissioning}
\label{sec:commissioning}

\subsection{Commissioning Milestones}
\label{ssec:commissioning-milestones}

> Commissioning work is being planned around two major milestones, ''First Photon`` and ''System First Light``. 

Author: ```suggestion
Commissioning work is being planned around two major milestones, ''First Photon`` and ''System First Light``. 
```

\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the LSSTCam focal plane to produce an image of the night sky.

> \textbf {System First Light}: Defined as the first science-grade images of the night sky obtained with LSSTCam. 

Reviewer 3: We have been working on a more detailed definition of System First Light https://sitcomtn-061.lsst.io/v/u-kbechtol-init/index.html

Our working definition has been that System First Light is achieved when we can "routinely acquire science-grade imaging across the full focal plane, and have a well understood technical path towards meeting the Construction Completeness criteria"

Author: Integrated into text

> \textbf {System First Light}: Defined as the first science-grade images of the night sky obtained with LSSTCam. 

Author: ```suggestion
\textbf {System First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the full focal plane (LSSTCam) and have a well understood technical path towards meeting the Construction Completeness criteria   \citeds{sitcomtn-061}.
```

Also referred to as \textbf{First Light}. 
These images will be shared with the worldwide community. 

First Photon occurs following the successful completion of system integration. 
There are no quality criteria applied to the images produced for the first photon milestone. 
First Photon  marks the start of the the on-sky engineering phase.
System First Light  marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases of commissioning.
The time from First Photon to System First Light is anticipated to be three months.
During these three months, the focus will be on fine tuning the system including optical alignment and improving the image quality, collecting calibration data, and carrying out \textit{First Look} science programs. 

> For a detailed description of all the commissioning milestones and the most current dates, see \citeds{dmtn-232}

Reviewer 3: ```suggestion
For a detailed description of all the commissioning milestones and the most current dates, see \citeds{dmtn-232}.
```

\subsection{Commissioning Schedule}
\label{ssec:commissioning-schedule}

Following the October 2023 project schedule workshop, the Rubin Construction Project is reoptimizing the sequence of integration activities based on important updates to projected subcomponent availability.

> Given recent delays in the Camera ship data and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 

Reviewer: data->date

Author: already fixed

> Given recent delays in the Camera ship data and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 

Author: ```suggestion
Given recent delays in the Camera ship date and important repairs needed to the summit dome crane, it will be beneficial to reinsert on-sky data taking with ComCam into the commissioning schedule for approximately two months, around July-August 2024. 
```

This will enable Telescope commissioning, primarily the Active Optics System, to occur approximately four months earlier than could be done with LSSTCam. 
With this updated strategy, System First Light with LSSTCam is projected to occur in January 2025, with Construction completion in May 2025.

> Without these changes, we predict an additional two months of delay.

Reviewer: I would delete this line. While true, It doesn't really make sense since of course we'd take longer if we don't adapt.

> Without these changes, we predict an additional two months of delay.

Author: ```suggestion

```

\begin{figure}[htb]
\centering
\includegraphics[width=0.98\linewidth]{figures/rubinobs_on-sky_commissioning_and_early_science.pdf}
\caption{Detailed schedule of commissioning  and early science activities relative to System First Light, as of October 2023.}
\label{fig:commissioning-es-schedule}
\vspace{0.1cm}
\end{figure}

Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
%Expected January 2025 - need to add the formula to code and  update the estimate here

> System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 

Reviewer: There is still some confusion about first photon. Maybe just avoid using that term. 

"System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 3 months after on sky commissioning activities begin."

Reviewer 2: The term First Photon is defined explicitly above at line 9 (with reference to LSSTCam).  Are you suggesting removing the term altogether from this document?  I thought it actually did a good job of de-confusing the situation, by providing a change-controlled reference for the term.

Author: No  but due to the fact that we do not know whether First Photon will be with ComCam or LSSTCam, rather than define an additional term to disambiguate FP with CC/LSSTCam, I agree with Bob that his suggestion is less confusing

Reviewer 2: Got it, thank you.

Reviewer 3: My understanding is that if we have ~2 months of on-sky observations with ComCam, the expected time between First Photon and System First Light with LSSTCam is reduced from 12 weeks to 7 weeks because we expect to have made some progress with AOS commissioning using ComCam.

Author: Changed SFL to ~3 months to ~2 months  post FP in the text

> System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 

Author: ```suggestion
```

> System First Light is currently expected in January 2025 (between XXX 2024 and YYY 2025) (\S~\ref{sec:timeline}), about 3 months after First Photon. 

Author: ```suggestion
System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 3 months after on sky commissioning activities begin.
```

The total amount of science validation time currently planned is about 8 weeks.  

> LSST data taking is expected to start about 4 months after System First Light.

Reviewer: With schedule uncertainty. LSST is 4-10 months after first light. 

"LSST data taking is expected to start 4-10 months after System First Light depending on construction schedue uncertainty and Operations readiness to start the survey following the start of full Survey Operations.

> LSST data taking is expected to start about 4 months after System First Light.

Reviewer: With schedule uncertainty. LSST is 4-10 months after first light.

"LSST data taking is expected to start 4-10 months after System First Light depending on construction schedue uncertainty and Operations readiness to start the survey following the start of full Survey Operations.

> LSST data taking is expected to start about 4 months after System First Light.

Author: ```suggestion
LSST data taking is expected to start 4-10 months after System First Light depending on construction schedule uncertainty and Operations readiness to start the survey following the start of full Survey Operations.
```

\subsection{Commissioning Schedule}
\label{ssec:commissioning-schedule}

Following the October 2023 project schedule workshop, the Rubin Construction Project is re-optimizing the sequence of integration activities in light of recent subcomponent delays, in particular, (i) delay in shipment of the Camera and (ii) necessary repairs to the  summit dome crane.
If current estimates hold, it will be beneficial to re-implement on-sky data-taking with ComCam (previously removed from the schedule in order to install LSSTCam sooner).

> The updated plan calls for on-sky data to be taken with ComCam for approximately two months, around July-August 2024 to support Telescope commissioning ,primarily the Active Optics System. 

Reviewer 3: ```suggestion
The updated plan calls for on-sky data to be taken with ComCam for approximately two months, around July-August 2024 to support Telescope commissioning, primarily the Active Optics System. 
```

This is approximately four months earlier than could be done with LSSTCam. 

\begin{figure}[htb]
\centering
\includegraphics[width=0.98\linewidth]{figures/rubinobs_on-sky_commissioning_and_early_science.pdf}
\caption{Detailed schedule of commissioning  and early science activities relative to System First Light, as of October 2023.}
\label{fig:commissioning-es-schedule}
\vspace{0.1cm}
\end{figure}

Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
System First Light is currently expected in January 2025 (\S~\ref{sec:timeline}), about 7 weeks after LSSTCam First Photon.
The total amount of science validation time currently planned is about 8 weeks.  

> LSST data taking is expected to start 4-10 months after System First Light depending on construction schedule uncertainty and Operations readiness to start the survey following the start of full Survey Operations.

Author: This is meant to reflect the fact that the start of the Operations era, i.e when responsibility is handed from the Construction project to Operations, is not necessarily the start of the survey

> LSST data taking is expected to start 4-10 months after System First Light depending on construction schedule uncertainty and Operations readiness to start the survey following the start of full Survey Operations.

Author: I'll remove the last part - that hopefully will make it clearer

The project schedule will continue to evolve as the remaining subcomponents are delivered. 
The final decision concerning ComCam on-sky data taking will be taken in February 2024.

\subsection{Commissioning Milestones}
\label{ssec:commissioning-milestones}

> Commissioning work is being planned around three major milestones, \textit{ComCam First Photon}, \textit{LSSTCam First Photon'} and \textit{System First Light}. 

Author: fixed

\textbf{ComCam First Photon}: The first image of the night sky produced by photons passing through the Rubin optical system and detected by the Commissioning Camera (ComCam).

\textbf {LSSTCam First Photon}: The first image of the night sky produced by photons passing through the Rubin optical system and detected by the LSST Science Camera (LSSTCam).

\textbf {System First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the LSSTCam full focal plane and have a well understood technical path towards meeting the Construction Completeness criteria   \citeds{sitcomtn-061}.
Also referred to as \textbf{First Light}. 

> These images will be shared with the worldwide community. 

Author: Removed it -- The topic requires more detail, that will be added later

\begin{figure}[htb]
\centering
\includegraphics[width=0.95\linewidth]{figures/commissioning-plan}

> \caption{Outline plan for the collection of commissioning data, as of October 2022.}

Reviewer: This figure needs to be updated

Reviewer 2: ```suggestion
\caption{Outline plan for the collection of commissioning data, as of October 2023.}
```
